docs/5-zarf-tutorials/0-creating-a-zarf-package.md

```diff
@@ -50,7 +50,7 @@ components:
   - name: wordpress  # specifies the name of our component and should be unique and unchanging through updates
     description: |   # (optional) a human-readable description of the component you are defining
       "Deploys the Bitnami-packaged WordPress chart into the cluster"
-    required: true   # (optional) sets the component as 'required' so that it is always deployed
+    optional: false   # (optional) sets the component as 'required' so that it is always deployed
```

Comment:
We should just remove this here since it isn't really needed to be introduces at this point - required is likely to be the desired default in most cases.